shared/mdparser.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/yuin/goldmark/renderer"
 	ghtml "github.com/yuin/goldmark/renderer/html"
 	"github.com/yuin/goldmark/util"
+    yaml "gopkg.in/yaml.v2"
```

Comment:
What's the preferred way of handling a type in a dependent package?

Reply:
Alright, I think I got it added as a direct dependeny using `go mod tidy`. Seems to have moved it to the top require (and added it's own indirect deps is what I'm assuming)